app-admin/systemrescuecd-x86/files/systemrescuecd.grub

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
```

Comment:
Is this your code or copied from somewhere? You should really add some copyright/copyleft here since it's a fair bit of code.